jax/experimental/jax2tf/jax2tf.py

```diff
@@ -1350,6 +1349,20 @@ def _eig(operand: TfVal, compute_left_eigenvectors: bool,
 
 tf_impl[lax_linalg.eig_p] = _eig
 
+def _eigh(operand: TfVal, lower: bool):
+  if operand.shape[-1] == 0:
+    return tuple([operand, tf.reshape(operand, operand.shape[:-1])])
+  if not lower:
+    operand = tf.linalg.adjoint(operand)
+  w, v = tf.linalg.eigh(operand)
+  dtype = operand.dtype
+  if dtype in [tf.complex64, tf.complex128]:
```

Comment:
Nit: this could be written as:
```
cast_type = { tf.complex64: tf.float32, tf.complex128: tf.float64 }.get(dtype)
if cast_type is not None:
  w = tf.cast(w, cast_type)
```